tests/testTriggers.go

```diff
@@ -177,6 +179,9 @@ var eventRing *ring.Ring
 //decommissionedNode for rejoin test
 var decommissionedNode = node.Node{}
 
+//node with auti pilot rule enabled
```

Comment:
nit: Add space after `//` and `node with auti pilot rule enabled`. Can we change the variable name to `autoPilotLabelNode`

Reply:
fixed